src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": [
```

Comment:
The Spotify API also needs to be defined here. Since we added the feature to play directly from Spotify Links.